rust/ql/src/queries/security/CWE-696/BadCtorInitialization.ql

```diff
@@ -40,16 +40,19 @@ class StdCall extends Expr {
 class PathElement = AstNode;
 
 query predicate edges(PathElement pred, PathElement succ) {
```

Comment:
Ah, so the fact it's a `query predicate` is relevant here.  We'll be generating the data anyway as we search for results, but if it's an intermediate predicate it will be discarded shortly after?

Reply:
Yes, `query` predicates will end up unfiltered in the `bqrs` result file.

Reply:
Updated.